pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -128,4 +128,8 @@ type ScaleDownConfig struct {
 	// How long a node should be unneeded before it is eligible for scale down
 	// +kubebuilder:validation:Pattern=([0-9]*(\.[0-9]*)?[a-z]+)+
 	UnneededTime *string `json:"unneededTime,omitempty"`
+
+	// Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down
+	// +kubebuilder:validation:Pattern=(0.[0-9]+)
```

Comment:
Is there any validation on this flag done by the cluster autoscaler? Can the threshold ever be set to `0` or `1`? As neither of these are currently allowed by this pattern

Reply:
Yes the validation is done in pkg/controller/clusterautoscaler/validator.go. (Cant seem to link the changes.) 

Since this threshold represents threshold below which scale down can be initiated. Setting threshold to 0 or 1 does not make much sense.

Reply:
i checked the autoscaler itself, and it does not have a validation for this that i could find, but i did see one unit test that was using a value above `1`. which makes me wonder if there is an over-provisioning case we need to account for?

basically, could a user set it to `1.5` with the expectations that the node could be over committed on the requested resources. i'm not sure if kube would allow this.

but, all that said, i'm not even sure the case for a value greater than `1` is valid. i'm fine with this regex the way it is.

Reply:
@SamuelStuchly here is where i saw the utilization threshold set over 1,

https://github.com/openshift/kubernetes-autoscaler/blob/master/cluster-autoscaler/cloudprovider/ovhcloud/ovh_cloud_node_group_test.go#L120